### ISSUE TITLE
Numeric format with quoted text is not formatting correctly.

### ISSUE THREAD

**Issue by jrsearles** [globalizejs/globalize#761]
When using the following format: '&#165;'#,##0.00;('&#165;'#,##0.00), it is throwing an error that the format is invalid. This appears to be happening because the pattern is being split on the semicolon for positive/negative patterns, ignoring the fact that the semicolon is in quotes.

This is happening i version v1.3.0.

**Comment by rxaviers** [comment]
Oh interesting... Seems like a bug. Can you use the utf-8 `¥` as a workaround?

**Comment by rxaviers** [comment]
EDIT: ... instead of the `&#165;` codeset?

**Comment by jrsearles** [comment]
Yeah, that's how i got around it in this case. I could see this being an issue with other literal text though, which is why i opened the ticket.
